From b3e770d3c2641de0e7f602527f426ab05f0e4171 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:22:13 -0400
Subject: [redhat] kabi: Adding symbol __nlmsg_put (net/netlink/af_netlink.c)

Message-id: <ca5b1a7b4452ba7730e4e86828653c76a70e1e7b.1410510773.git.poros@redhat.com>
Patchwork-id: 95922
O-Subject: [PATCH RHEL7.1 013/233] kabi: Adding symbol __nlmsg_put (net/netlink/af_netlink.c)
Bugzilla: 1077335
RH-Acked-by: Jiri Benc <jbenc@redhat.com>

Bugzilla: 1077335
    https://bugzilla.redhat.com/show_bug.cgi?id=1077335

    Description:
    ============
    Netlink API.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/__nlmsg_put
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/__nlmsg_put
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/__nlmsg_put

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/__nlmsg_put b/redhat/kabi/kabi-module/kabi_ppc64/__nlmsg_put
new file mode 100644
index 0000000..8da483c
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/__nlmsg_put
@@ -0,0 +1,2 @@
+#1-
+0x6bd99593 __nlmsg_put vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/__nlmsg_put b/redhat/kabi/kabi-module/kabi_s390x/__nlmsg_put
new file mode 100644
index 0000000..66ac369
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/__nlmsg_put
@@ -0,0 +1,2 @@
+#1-
+0x694d4ab6 __nlmsg_put vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/__nlmsg_put b/redhat/kabi/kabi-module/kabi_x86_64/__nlmsg_put
new file mode 100644
index 0000000..66fce8a
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/__nlmsg_put
@@ -0,0 +1,2 @@
+#1-
+0xfa31c478 __nlmsg_put vmlinux EXPORT_SYMBOL
-- 
1.7.1